core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/DataRepoExpressionFixture.scala

```diff
@@ -116,10 +155,26 @@ trait DataRepoExpressionFixture {
 
   val invalidOutputExpressions: Seq[String] = unparseableOutputExpressions ++ invalidEntityOutputExpressions
 
+  /** TABLES AND COLUMNS */
   val defaultFixtureRootTableName = "rootTable"
   // These should reflect the columns used in validEntityInputExpressions above
-  val defaultFixtureRootTableColumns = List("gvcf", "with-dash", "library:cohort", "library:cohort1", "arbitrary:whatever", "underscores_are_ok", "_", "case_sample")
+  val defaultFixtureRootTableColumns = List("gvcf", "with-dash", "underscores_are_ok", "_", "case_sample")
   val defaultFixtureTables: List[TableModel] = List(
     new TableModel().name(defaultFixtureRootTableName).columns(defaultFixtureRootTableColumns.map(new ColumnModel().name(_)).asJava)
   )
-}
\ No newline at end of file
+
+  val rootTableName = "rootTable"
+  val rootTableColumns = List("root_table_column", "gvcf", "with-dash", "underscores_are_ok", "_", "case_sample", "root_second")
+
+  val secondTableName = "secondTable"
+  val secondTableColumns = List("second_table_column", "second_third", "second_root")
```

Comment:
If possible, would like to rename these columns again, as I struggle with this naming convention